intro-HPC/ch_preparing_the_environment.tex

```diff
@@ -464,6 +757,21 @@ \subsection{Activating and de-activating modules}
 
 This will load the two modules as well as their dependencies.
 
+\ifgent
+Finally, to switch between clusters you can swap the ``cluster'' modules.
+These are special modules that change what modules are available for you,
+and what cluster your jobs will be queued in.
+By default you are working on \hpcname. To switch to e.g. raichu you need:
+
+\begin{prompt}
+%\shellcmd{module swap cluster/raichu}%
+\end{prompt}
+
+This will switch the cluster you submit your jobs to (with qsub) to raichu
+and only show you modules that are installed on raichu when you run
+``module av''.
```

Comment:
this is already explained in `\inputsite{available-modules}`, see line 665 in this file